plugins/bkpr/onchain_fee.c

```diff
@@ -260,10 +286,10 @@ static void insert_chain_fees_diff(struct bkpr *bkpr,
 		if (!bitcoin_txid_eq(&ofs[i]->txid, txid))
 			continue;
 		if (!amount_msat_accumulate(&current_amt, ofs[i]->credit))
-			db_fatal(bkpr->db, "Overflow when adding onchain fees");
+			plugin_err(cmd->plugin, "Overflow when adding onchain fees");
```

Comment:
the recorder tests pass in NULL for `cmd`, which would blow this up... guess we don't test for this :)

Reply:
Yes, since this Shouldn't Happen.... since the result is supposed to be a different kind of crash, I don't think we care!